### ISSUE TITLE
option to include memtest on iso

### ISSUE THREAD

**Issue by FlorianHeigl** [cobbler/cobbler#1476]
Hi,

as far as I could see in my search: even if you add a memtest image, it'll only presented as a pxe option.
it is small and should by included in the generated iso for sheer helpfulness...

Oldest request dates to 2008 apparently :)
https://lists.fedorahosted.org/pipermail/cobbler/2008-December/001869.html

**Comment by alanoe** [comment]
Please provide context when opening an issue.

You are talking about the ISO Cobbler created with cobbler buildiso (CLI command) and which contains all the distro installations configurations available in all profiles and systems in a Cobbler server (or a subset of them)?

**Comment by FlorianHeigl** [comment]
Yes, it's about cobbler buildiso. The original feature request had it described nicely. Could I ask you to please just read it carefully, I took hours to find the exact status quo (including when the ball was dropped, rediscussed in 2012, etc, etc) and this is exact first useful 'primary' post on this.
